packages/devtools_app/lib/src/service_extensions.dart

```diff
@@ -139,7 +139,7 @@ final profileWidgetBuilds = ToggleableServiceExtensionDescription<bool>._(
 final repaintRainbow = ToggleableServiceExtensionDescription<bool>._(
   extension: 'ext.flutter.repaintRainbow',
   description: 'Repaint Rainbow',
-  icon: createImageIcon('icons/repaint_rainbow.png'),
+  icon: createImageIcon('icons/repaint_rainbow@2x.png'),
```

Comment:
you don't need to add the 2x. Unless there is a bug in the flutter version, this icon code is designed to automatically use the 2x versions on a high dpi devices like the matching intellij code does.

Reply:
I don't believe it's working - from local testing, it does look like there's less pixelation when choosing the 2x paths.

I believe that the `foo@wx.png` convention is what IntelliJ uses, and flutter uses something like `.../2.0x/my_icon.png` (i.e., based on directory name).

Reply:
I opened https://github.com/flutter/devtools/issues/2072 to track addressing this.